helix-term/src/keymap.rs

```diff
@@ -250,6 +251,14 @@ impl Keymap {
         &self.root
     }
 
+    /// Returns list of keys waiting to be disambiguated.
+    pub fn pending(&self) -> Option<&Vec<KeyEvent>> {
```

Comment:
&Vec<T> is an antipattern, always prefer &[T]. Here you should also just return the slice, then use `is_empty()` in your code.

Reply:
```suggestion
    pub fn pending(&self) -> Option<&[KeyEvent]> {
```

Reply:
Fixed 👍🏾